### PR TITLE
added better wrap support to LeftRightArranger

### PR DESCRIPTION
**Pull request by dmikey**
Enyo-DCO-1.1-Signed-off-by: Derek Anderson <toxigenicpoem@gmail.com>

the current left and right arranger gets direction handling mixed up when index values change from Finish to End, and End to Finish. 

Added variable 'w' which will check to see if the rotator is 'wrapping' during arrangement.

### CODE REVIEW

**Comment by aarontam** [comment]
@dmikey Do you think you can fix the merge conflict? Thanks!

File: panels/source/arrangers/OtherArrangers.js [hunk 1]
```diff
@@ -45,9 +45,12 @@ enyo.kind({
 			var f = this.container.toIndex;
 			var c$ = this.getOrderedControls(f);
 			var o = Math.floor(c$.length/2);
+		        var wf = (s === 0 && f == (c$.length-1));
```
Comment (aarontam) [inline panels/source/arrangers/OtherArrangers.js]: Does `wf` refer to "forwards" and `wb` refer to "backwards"? If so, should these be reversed? I don't think it makes a difference in the ultimate logic, but more for semantics.

Comment (dmikey) [inline panels/source/arrangers/OtherArrangers.js]: when you say reversed, you mean? Line numbers? 

wf "wrap forward", wb "wrap backward" determines if 0 panel to max panel, or max panel to 0 panel flow happened.

Comment (aarontam) [inline panels/source/arrangers/OtherArrangers.js]: Yep - I was wondering if it should instead be:
```
var wb = (s === 0 && f == (c$.length-1)); // we're moving backward, from the first panel to the last panel
var wf = (s == (c$.length-1) && f === 0); // we're moving forward, from the last panel to the first panel
```
Maybe our definitions of wrapping forward/backward are different. :)

File: panels/source/arrangers/OtherArrangers.js [hunk 2]
```diff
@@ -45,9 +45,12 @@ enyo.kind({
...
+		        var wb = (s == (c$.length-1) && f === 0);
+		        var w = !wb ? wf ? wf : s > f : !wb;
```
Comment (aarontam) [inline panels/source/arrangers/OtherArrangers.js]: Using the current variable names can we simplify this as:
```
var w = !wb && (wf || s > f);
```

Comment (dmikey) [inline panels/source/arrangers/OtherArrangers.js]: Awesome!

**Comment by dmikey** [comment]
Closing, looking again - this doesn't look like it is currently fixing what I originally intended for it to fix.

**Pull request closed without merging**